index.html

```diff
@@ -29,34 +29,168 @@ <h2 class="title ocupation">
+          My Recent Works
+        </h2>
+        <div class="card-container">
+          <div class="card">
+            <div class="card-img">
+              <p>img</p>
+            </div>
+            <div class="card-content">
+              <h4 class="p-title"> 
+                Multi-Post Stories Gain+Glory 
+              </h4>
+              <ul class="tags">
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>JavaScript</li>
+                <li>html</li>
+              </ul>
+              <button type="button" class="primary-btn">
+                See Project
+              </button>
+            </div>
+          </div>
+          <div class="card">
+            <div class="card-img">
+              <p>img</p>
+            </div>
+            <div class="card-content">
+              <h4 class="p-title"> 
+                Multi-Post Stories Gain+Glory 
+              </h4>
+              <ul class="tags">
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>JavaScript</li>
+                <li>html</li>
+              </ul>
+              <button type="button" class="primary-btn">
+                See Project
+              </button>
+            </div>
+          </div>
+          <div class="card">
+            <div class="card-img">
+              <p>img</p>
+            </div>
+            <div class="card-content">
+              <h4 class="p-title"> 
+                Multi-Post Stories Gain+Glory 
+              </h4>
+              <ul class="tags">
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>JavaScript</li>
+                <li>html</li>
+              </ul>
+              <button type="button" class="primary-btn">
+                See Project
+              </button>
+            </div>
+          </div>
+          <div class="card">
+            <div class="card-img">
+              <p>img</p>
+            </div>
+            <div class="card-content">
+              <h4 class="p-title"> 
+                Multi-Post Stories Gain+Glory 
+              </h4>
+              <ul class="tags">
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>JavaScript</li>
+                <li>html</li>
+              </ul>
+              <button type="button" class="primary-btn">
+                See Project
+              </button>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="about">
+        <div class="about-container">
+          <h2>
+            About me
+          </h2>
+          <img src="/images/about.svg" alt="about">
+        </div>
+        <p>
+          I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <img class="about2" src="/images/about2.svg" alt="about2">
```

Comment:
- Please make this a background image with the CSS property background-image and position appropriately,  as it is not a main part of the portfolio but is for decoration. Do the same for other images that are for decoration or background-images.